.gitignore

```diff
@@ -30,3 +30,5 @@ gradle-app.setting
 
 # Cache of project
 .gradletasknamecache
+
+.idea
```

Comment:
I'm not a fan of adding editor-specific things to each project `.gitignore` file. I'd suggest to use a global `.gitignore` file for these things instead. (see https://help.github.com/articles/ignoring-files/#create-a-global-gitignore)

Reply:
Thanks, I didn't know that a global ignore was possible. I removed the gitignore change.